jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>The cell-type specific response of perturbations is, imo, too broad and not-well understood (is it a difference in target protein? something else entirely??). I would shorten this paragraph to just the first two sentences and state that Augur give one a way to quantify the degree of response.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>